Dockerfile.dapper

```diff
@@ -1,30 +1,20 @@
-FROM ubuntu:16.04
-# FROM arm=armhf/ubuntu:16.04 arm64=aarch64/ubuntu:16.04
+FROM registry.suse.com/bci/golang:1.19
 
 ARG DAPPER_HOST_ARCH
-ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH}
+ENV ARCH $DAPPER_HOST_ARCH
 
-RUN apt-get update && \
-    apt-get install -y gcc ca-certificates git wget curl vim less file make && \
-    rm -f /bin/sh && ln -s /bin/bash /bin/sh
+RUN zypper -n install git docker vim less file curl wget
+#RUN go install golang.org/x/lint/golint@latest && go install golang.org/x/tools/cmd/goimports@latest
+#RUN if [[ "${ARCH}" == "amd64" ]]; then \
```

Comment:
do we need the commented parts or can be removed?

Reply:
Removed. Thanks for spotting this.